scripts/test/runtests.sh

```diff
@@ -16,6 +16,14 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 source "$DIR/../common/_common.sh"
 
+export StartPath=$PATH
```

Comment:
I noticed the compile scripts did it so I also did for sanity,  consistence, and avoiding to google it :).
Will fix, thanks.